client/context/context.jsx

```diff
@@ -132,6 +146,10 @@ export const AppProvider = ({ children }) => {
   return (
     <AppContext.Provider
       value={{
+<<<<<<< HEAD
```

Comment:
Oops thought I saw a check running and thought this got resolved. Fix this any you can merge

Reply:
thank u for checking carefully. Had a rush morning when sending PR.